src/main/kotlin/com/wout/weather/service/WeatherService.kt

```
@@ -32,139 +33,206 @@ class WeatherService(
 
-    /**
-     * 도시명 기반 WeatherData 엔티티 조회 (WeatherScoreService용)
-     */
-    fun getCurrentWeatherDataByCity(cityName: String): WeatherData {
-        return weatherRepository.findLatestByCityName(cityName)
-            ?: throw ApiException(
-                WEATHER_DATA_NOT_FOUND,
-                WEATHER_DATA_NOT_FOUND.message
-            )
+    private fun formatTemperature(temperature: Double): Double {
+        return String.format("%.1f", temperature).toDouble()
```

detekt (1.23.8)

[warning] String.format("%.1f", temperature) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)